apis/api-media/src/workers/bigQuery/importers/masterUrls/masterUrls.spec.ts

```
@@ -0,0 +1,130 @@
+import { VideoVariant } from '.prisma/api-media-client'
```

ESLint

[error] Useless path segments for ".prisma/api-media-client", should be "./.prisma/api-media-client"

(import/no-useless-path-segments)